be/src/vec/sink/writer/vtablet_writer_v2.cpp

```diff
@@ -568,23 +594,39 @@ Status VTabletWriterV2::close(Status exec_status) {
 
         // send CLOSE_LOAD on all non-incremental streams if this is the last sink
         if (is_last_sink) {
-            RETURN_IF_ERROR(_load_stream_map->close_load(false));
+            auto st = _load_stream_map->close_load(false);
+            if (!st.ok()) {
```

Comment:
We should handle the error here. The failure of downstream nodes should not be visible to VtabletWriter.